user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -51,7 +54,75 @@ function commonFunctions:isMediaApp()
   end
 
   return isMedia
+end
+
+function commonFunctions:waitForSDLStart(test)
+  local step = 100
```

Comment:
@LitvinenkoIra add comment, that you check that SDL opened all ports each 100 ms.